### PR TITLE
Remove ARCH specific apt repo info to enable arm support

### PR DESCRIPTION
**Pull request by hh**

### CODE REVIEW

File: images/base/Dockerfile [hunk 1]
```diff
@@ -75,17 +71,18 @@ ARG DOCKER_VERSION="18.06.*"
...
     && clean-install "docker-ce=${DOCKER_VERSION}"
 
 # Install CNI binaries to /opt/cni/bin
 # TODO(bentheelder): doc why / what here
 ARG CNI_VERSION="0.6.0"
-ARG CNI_TARBALL="cni-plugins-${ARCH}-v${CNI_VERSION}.tgz"
 ARG CNI_BASE_URL="https://storage.googleapis.com/kubernetes-release/network-plugins/"
-ARG CNI_URL="${CNI_BASE_URL}${CNI_TARBALL}"
-RUN curl -sSL --retry 5 --output /tmp/cni.tgz "${CNI_URL}" \
+RUN export ARCH=$(dpkg --print-architecture) \
```
Comment (BenTheElder) [inline images/base/Dockerfile]: neat!

**Comment by BenTheElder** [comment]
changes LGTM, can you squash this to remove the merge commit and run `hack/update-generated.sh`?

**Comment by alejandrox1** [comment]
cla wise there are two authors to this PR. @devaii have you signed the CLA as well?

Pull-kind-verify failing with:
```
verifying generated ...
unexpectedly dirty working directory after hack/update-generated.sh
...

One or more verify checks failed! See output above...
```
So as Ben commented, need to run `hack/update-generated.sh`.

Pull-kind-performance on the other hand should be fixed by #335 (?)
```
W0223 14:17:25.415] Error: error building node image: failed to copy build artifact: lstat /go/src/k8s.io/kubernetes/bazel-bin/build/debs/kubectl.deb: no such file or directory
W0223 14:17:25.418] + cleanup
W0223 14:17:25.418] + kind export logs /workspace/_artifacts/logs
W0223 14:17:25.547] Error: unknown cluster "kind"
W0223 14:17:25.550] + true
W0223 14:17:25.550] + [[ '' = true ]]
W0223 14:17:25.551] + rm -f _output/bin/e2e.test
W0223 14:17:25.552] + [[ -n /tmp/tmp.VVaqHZHqxJ ]]
W0223 14:17:25.552] + rm -rf /tmp/tmp.VVaqHZHqxJ
W0223 14:17:25.567] Traceback (most recent call last):
W0223 14:17:25.568]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 50, in <module>
W0223 14:17:25.568]     main(ARGS.env, ARGS.cmd + ARGS.args)
W0223 14:17:25.568]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 41, in main
W0223 14:17:25.568]     check(*cmd)
W0223 14:17:25.568]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 30, in check
W0223 14:17:25.568]     subprocess.check_call(cmd)
W0223 14:17:25.568]   File "/usr/lib/python2.7/subprocess.py", line 186, in check_call
W0223 14:17:25.568]     raise CalledProcessError(retcode, cmd)
W0223 14:17:25.569] subprocess.CalledProcessError: Command '('bash', '-c', 'cd ./../../k8s.io/kubernetes && ./../../sigs.k8s.io/kind/hack/ci/e2e.sh')' returned non-zero exit status 1
E0223 14:17:25.573] Command failed
I0223 14:17:25.573] process 703 exited with code 1 after 2.5m
E0223 14:17:25.573] FAIL: pull-kind-conformance-parallel
```

**Comment by devaii** [comment]
I have now signed the CNCF CLA

**Comment by dims** [comment]
@devaii i kicked the tires, looks good. can you please fix the verify error?

**Comment by dims** [comment]
cc @vielmetti

**Comment by dims** [comment]
@hh @devaii looks like #358 got merged. So let's close this out.

/close

**Comment by BenTheElder** [comment]
Thanks again all!

**Pull request closed without merging**